.github/workflows/ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     - name: Install npm dependencies and build
       run: |
         npm install
-        export BASE_PATH="/aiida-registry/pr-preview/pr-${{ github.event.number }}/"
+        export VITE_BASE_PATH="/aiida-registry/pr-preview/pr-${{ github.event.number }}/"
```

Comment:
The environment variable is for pr-preview only? What is after the pr merged to the main, does this path needs to be changed?